devlib/target.py

```diff
@@ -70,6 +71,8 @@ def connected_as_root(self):
     @property
     @memoized
     def is_rooted(self):
+        if self.conn.connected_as_root:
+            return True
```

Comment:
Does this need to be set here? 
If so, the android implementation needs to be updated to be consistent.